.gitignore

```diff
@@ -49,4 +49,7 @@ dist*
 /bazel-*
 
 # Direnv
-.envrc.local
\ No newline at end of file
+.envrc.local
+
+# Stack
+.stack-work
```

Comment:
This is probably worth keeping, even once you remove your stack commit from the PR.